### DISCUSSION TITLE
i dont pretty much like about the tarantula, maybe re add the legacy spider and its behaviour

### DISCUSSION THREAD

**Discussion by TYAGAGerson** [SuperTux/supertux#3026] [Q&A] [answered]
the tarantula is pretty much unrelated in style and is really creepy(both the new tarantula and the spidermite) especially the eyes that looks horrifying. i like the legacy behaviour because its easy to predict. i mean just readd not replace the new behaviour maybe at the type option or something. sorry if my english is not too good.

**Comment by tylerandari13** [comment, marked as answer]
I guess i could understand the creepiness (although im not sure of an answer) but in terms of the behavior the new spidermite outclasses the old one by a long shot. Being able to place him on the ceiling and have him fall down on his web looks much cooler than just... moving up and down... It also has more potential to be combined with other elements to create fun experiences. The old spidermite was nothing more than a glorified flying snowball, in fact, if you want the old spidermite behavior reskinning a flying snowball is probably the way to go to get it back.

**Reply by TYAGAGerson** [reply]
ok, that flying snowball is pretty much up and down but very random. i think i will use the old sprite/graphics for the spidermate unless i know how to make a stompable platform with the scripted objects.
